### ISSUE TITLE
CurlWrapper.php

### ISSUE THREAD

**Issue by Larphoid** [aineko-m/iveeCrest#1]
o/

This is not an issue but just a remark about something I noticed in you'r getrefreshtoken.php.
I downloaded it to use as an example for CREST authentication. The line **rtrim($fields_string, '&');** works,  but it is redundant as you don't do anything with the value returned. **rtrim** returns the resulting string, it doesn't change the string passed to it in the first parameter. Obvioulsy it doesn't seem to matter for the authentication process. Probably because that last (empty) **&** will just be ignored by the receiving party (CREST in this case).
By the way, thanks for posting getrefreshtoken.php ! Was searching for some kind of example on CREST authentication and this was about the only thing I could find.

**Comment by aineko-m** [comment]
You are right of course. I haven't put much time into this project for a while (I have yet to start playing with ESI), but I'll fix the issue at some point. Happy to have helped someone :)

**Comment by aineko-m** [comment]
Oh and btw, you are looking at an old version of getrefreshtoken, as iveeCrest got merged into [iveeCore](https://github.com/aineko-m/iveeCore/blob/master/www/getrefreshtoken.php).
